src/webview/__tests__/generateInboundEventEditSequence-test.js

```diff
@@ -198,7 +198,9 @@ const pmMessages6 = [
 const baseState = eg.reduxStatePlus({
   streams: [...eg.plusReduxState.streams, stream1, stream2],
   subscriptions: [
-    ...eg.plusReduxState.subscriptions,
+    // Could spread eg.plusReduxState.subscriptions, for completeness…but
+    // then we'd invite flakes where random stream IDs in there sometimes
+    // collide with our fixed ones, changing whether a stream is subscribed
     eg.makeSubscription({ stream: stream1 }),
     eg.makeSubscription({ stream: stream2 }),
```

Comment:
Do we have any fixed streamIDs here other than those of stream1 and stream2?

I think we don't. That was also the idea of https://github.com/zulip/zulip-mobile/commit/932c28dd0571c42c9ec5974ab39508fea52144c8 , which was to make the streams always subscribed (except in the one test where we specifically want them unsubscribed.)